devtools/platform-descriptor-json/src/main/java/io/quarkus/platform/descriptor/loader/json/impl/QuarkusJsonPlatformDescriptorLoaderImpl.java

```diff
@@ -19,11 +20,12 @@ public QuarkusJsonPlatformDescriptor load(final QuarkusJsonPlatformDescriptorLoa
         final QuarkusJsonPlatformDescriptor platform = context
                 .parseJson(is -> {
                     try {
-                        ObjectMapper mapper = new ObjectMapper()
-                                .enable(JsonParser.Feature.ALLOW_COMMENTS)
-                                .enable(JsonParser.Feature.ALLOW_NUMERIC_LEADING_ZEROS)
+                        ObjectMapper mapper = JsonMapper.builder()
+                                .enable(JsonReadFeature.ALLOW_JAVA_COMMENTS)
+                                .enable(JsonReadFeature.ALLOW_LEADING_ZEROS_FOR_NUMBERS)
                                 .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-                                .setPropertyNamingStrategy(PropertyNamingStrategy.KEBAB_CASE);
+                                .build()
+                                .setPropertyNamingStrategy(PropertyNamingStrategies.KEBAB_CASE);
```

Comment:
I'm surprised you apply this one after the `build()` if the `ObjectMapper` is supposed to be immutable?

Reply:
it's gonna be immutable in version 3

This is PR is handling just the deprecated constants, it's not about replacement of `new ObjectMapper`, there are around 70 occurrences of `new ObjectMapper` in Quarkus codebase.

I didn't find a way to set property naming strategy as part of the builder command :/ I can try to search again and a bit more.

Reply:
Was just looking in `JsonMapper.Builder`, but didn't check `MapperBuilder`, will update the PR shortly.